cms/models.py

```diff
@@ -94,10 +146,25 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class LearningAreasPage(BootcampProductChildPage):
```

Comment:
Could you please rename this and related classes similar to "ThreeColumnSection", "ThreeColumnBlock" etc? I think we had the same issue in xPRO where we started off naming these classes after where they were going to be used like "WhoShouldLearn", "LearningOutcomes" etc and then we had to rename them to "TextImageSection", "TextVideoSection" etc. because they needed to be reused on the homepage.